package-lock.json

```diff
@@ -38920,18 +38920,18 @@
       "devDependencies": {
         "@clerk/eslint-config-custom": "*",
         "@cloudflare/workers-types": "^3.18.0",
-        "@types/chai": "^4.3.16",
+        "@types/chai": "^4.3.3",
```

Comment:
@BRKalow can you review that these changes are ok since it seems unexpected for these changes to be included in this pr?

Reply:
Weird, no idea why this happened

Reply:
this is happening on main as well, nothing done in this pr

Reply:
Renovate update. Got confirmation this is good to go.